app/views/product_users/index.html.haml

```diff
@@ -20,22 +20,17 @@
         = text("import_button")
         %span{ style:"display:none" }= f.file_field :file, onchange: "form.submit()"
 
-- if @product_users.nil?
-- elsif @product_users.empty?
-  %p.notice= text("none")
```

Comment:
I moved it to users_results partial. I had to add `if @search` because depending on that we should say that there aren't any approved users or that the search results are empty. I didn't find a message I liked and that fitted well both cases

Reply:
got it - I left a comment on the users_search partial